src/components/OwnerRegForm.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { View, Text, TextInput, StyleSheet, Button, TouchableOpacity } from 'react-native'
+
+
+const UserRegForm = () => {
+    const [userName, setUserName] = useState('');
+    const [password, setPassword] = useState('');
+    const [email, setEmail] = useState('')
+    const [truckName, setTruckName] = useState('')
+    return (
+        <View>
+            <Text>User Name</Text>
+            <TextInput style={styles.input}
+                placeholder="User Name"
+                value={userName}
+                onChangeText={text => setUserName(text)}
+            />
+            <Text>Email</Text>
+            <TextInput placeholder="Email"
+                value={email}
+                onChangeText={text => setEmail(text)}
+            />
+            <Text>Name of foodtruck</Text>
+            <TextInput placeholder="Truck Name"
+                value={truckName}
+                onChangeText={text => setTruckName(text)}
+            />
```

Comment:
I think  we might want to have the truck creation a separate screen that loads after the owner registers, so that more info can be added (truck name, food type, do they do private catering, their home location, etc.) Also - if an owner has more than one truck it would be better separated from the account reg form.